sources/packages/web/src/views/aest/Login.vue

```diff
@@ -73,6 +74,7 @@ export default defineComponent({
   setup() {
     const { executeLogin } = useAuth();
     const login = async () => {
+      sessionStorage.setItem(USER_LOGIN_TRIGGERED, "true");
```

Comment:
It can use the `static` defined `userLoginTriggered`. Same for the other login files.